### ISSUE TITLE
[BUG Report] checkout error

### ISSUE THREAD

**Issue by sdleiw** [Sylius/ShopApiPlugin#241]
Hi, 

I could not make `PUT /checkout/{cartid}/address` call after updating my local api server, got a 500 error

Type error: Return value of Sylius\\Bundle\\CoreBundle\\Checkout\\CheckoutResolver::getRequestedGraph() must be of the type string, null returned"

have noticed that in v1.0.3 update the checkout_resolve config was removed, but it was suggested  for the shop-api server in the readme of this repo, can someone take a look at that, by the way, with this config the put address API call works

my current api server dependencies
- sylius v1.0.3
- syliusShopApiPlugin dev-master
- nelmio/cors-bundle v1.5.3

**Comment by Hailong** [comment]
Hi @sdleiw , I saw the same error when loading the checkout page, it might be the same issue for API. Changing the path matching pattern will resolve the problem, you could check it out from my PR here: https://github.com/Sylius/SyliusShopApiPlugin/pull/246

**Comment by bochkarev-artem** [comment]
The same issue with version sylius 1.6.1 and sylius/shop-api-plugin v1.0.0-rc.2
The documentation of [this mr](https://github.com/Sylius/ShopApiPlugin/pull/246) helped
Although works also with this configuration:
`sylius_shop:
    checkout_resolver:
        pattern: "%sylius.security.shop_regex%/checkout/.+"`

**Comment by bochkarev-artem** [comment]
i would sugget reopenning the issue, or updating documentation

**Comment by misnard** [comment]
> The same issue with version sylius 1.6.1 and sylius/shop-api-plugin v1.0.0-rc.2
> The documentation of [this mr](https://github.com/Sylius/ShopApiPlugin/pull/246) helped
> Although works also with this configuration:
> 
> ```yaml
> sylius_shop:
>     checkout_resolver:
>         pattern: "%sylius.security.shop_regex%/checkout/"
> ```

I found this config inside of this file config/packages/_sylius.yaml

Thanks.

**Comment by mamazu** [comment]
Is this still an issue in the current configuration? Because now there is no config key like `checkout_resolver` in the `README` anymore.

**Comment by misnard** [comment]
Yes, i face this problem and i didn't found any help in README hopefully this topic help me with some research.

**Comment by mamazu** [comment]
This seems to be still an issue.

**Comment by mamazu** [comment]
This seems to be still an issue.

**Comment by alexander-schranz** [comment]
Can confirm that it still exist.

Adopting the sylius configuration the following way seems to help:

https://github.com/Sylius/ShopApiPlugin/pull/246/files

Why was this removed from the README?

**Comment by lchrusciel** [comment]
Isn't it related to #647?

**Comment by alexander-schranz** [comment]
@lchrusciel It could the error was something like the `graph` is not defined but not sure.

**Comment by FDiskas** [comment]
Same problem after adding suggested changes - works
